department-of-veterans-affairs/va.gov-cms
# VAGOV-5835: Remove unused legacy flags.

## What this does
 - Whittles down our flags list to 3 still in use in vets-website (said repo has already been cleaned, so it is safe to merge this pr).

## To Test
 - Go to `admin/config/system/feature_toggle` - you should only see three (disabled) flags.
